src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -268,21 +269,15 @@ function RemoteFunctions(experimental) {
                 "box-shadow": "0 0 1px #fff",
                 "box-sizing": "border-box"
             };
+            
+            var mergedStyles = Object.assign({}, stylesToSet,  config.remoteHighlight.stylesToSet);
```

Comment:
I think Object.assign is not available on Linux (which uses Chrome 29). Could you try lodash assign or extend instead.